package.json

```diff
@@ -33,6 +34,7 @@
     "start": "concurrently npm:start:remote-1 npm:start:remote-2 npm:start:shell",
     "//build": "surge redirects all unmatched routes to 200.html, so copy index.html to 200.html to support client-side rendering of deep links.",
     "build": "webpack --mode production && cd dist && cp index.html 200.html",
+    "build:dev": "cross-env NODE_ENV=development webpack --config webpack.config.js",
```

Comment:
Why this change?

Reply:
I added this because I needed to build a dev container.  I was going to make a docker-compose solution, so I just added a dev snippet for cross-env to declare development mode for webpack so we can get the localhost port configurations.  There is a bool operator inside the web pack configurations `isProduction`